src/views/Landing.vue

```diff
@@ -0,0 +1,1172 @@
+<!-- eslint-disable vue-i18n/no-raw-text -->
+<template>
+  <div class="landing-page-container">
+    <header id="header" class="sh-header">
```

Comment:
it won't break anything here, but in general, we shouldn't refer by id except the root node (#app)